genesyscloud/architect_ivr/resource_genesyscloud_architect_ivr.go

```diff
@@ -84,7 +84,8 @@ func readIvrConfig(ctx context.Context, d *schema.ResourceData, meta interface{}
 		}
 
 		_ = d.Set("name", *ivrConfig.Name)
-		_ = d.Set("dnis", lists.StringListToSetOrNil(ivrConfig.Dnis))
+		dnis := lists.Map(*ivrConfig.Dnis, util.FormatAsCalculatedE164Number)
```

Comment:
Is there any risk of seeing a nil pointer dereference error here? Maybe we should check if `ivrConfig.Dnis` is nil first